modules/utxo-lib/test/networks.ts

```diff
@@ -41,7 +42,13 @@ describe('networks', function () {
       isLitecoin(network));
 
   const bitcoinTestnetSharedScriptPrefix = (network) =>
-    bitcoinTestnetSharedPubkeyPrefix(network) && !isLitecoin(network);
+    // bitcoinTestnetSharedPubkeyPrefix(network) && !isLitecoin(network);
+    isTestnet(network) &&
+    (isBitcoin(network) ||
+      isBitcoinCash(network) ||
+      isBitcoinGold(network) ||
+      isBitcoinSV(network) ||
+      isDogecoin(network));
```

Comment:
while you're at it, maybe make them the same order as `bitcoinTestnetSharedPubkeyPrefix`?

Reply:
changed the order in `bitcoinTestnetSharedPubkeyPrefix` to meet correct alphabetical order.